packages/main/src/TableGrowing.ts

```diff
@@ -147,6 +147,10 @@ class TableGrowing extends UI5Element implements ITableGrowing {
 	}
 
 	onTableAfterRendering(): void {
+		if (this.type === TableGrowingMode.Scroll && !this._table) {
+			this._invalidateTable();
+		}
```

Comment:
I am a little confused Duc.

If `this._table` is not yet set how `this._invalidateTable();`is going to work? I guess these lines does nothing but then how tests are covering this?

you might probably set this._table in line 151 but can we somehow let the lines 168-170 handle this?